### PR TITLE
added bookmark service to see bookmark exisit

### PR DESCRIPTION
**Pull request by ujjvalksharma**
added bookmark service to see bookmark exisit

### CODE REVIEW

File: controllers/BookmarkController.ts [hunk 1]
```diff
@@ -64,7 +68,8 @@ export default class BookmarkController implements BookmarkControllerI {
   bookmarkATuit = (req: Request, res: Response) =>
```
Comment (Zevez7) [inline controllers/BookmarkController.ts:68]: You need a js doc

File: daos/BookmarkDao.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ export default class BookmarkDao implements BookmarkDaoI {
   private constructor() {}
 
   async bookmarkATuit(tid: string, uid: string): Promise<Bookmark> {
```
Comment (Zevez7) [inline daos/BookmarkDao.ts:29]: You need a js doc for all the functions

**Review by Zevez7 — CHANGES_REQUESTED** [review]
Please post your bookmark postmen test so I can test the server.

File: daos/BookmarkDao.ts (hunk 2 above)
Comment (Zevez7) [reply daos/BookmarkDao.ts:29]: Is there a need for you to most of the package-lock.json? Would that affect later install of the npm?

Comment (ujjvalksharma) [reply daos/BookmarkDao.ts:29]: There is no need to add js doc in functions that are implementation of an interface.

Comment (ujjvalksharma) [reply daos/BookmarkDao.ts:29]: Merging as no changes are needed

Comment (ujjvalksharma) [reply daos/BookmarkDao.ts:29]: Post man collection for bookmarks:

get all bookmark of a user: http://localhost:5002/users/638f5daf5561b563b87b7577/bookmarks
Create bookmark: 
http://localhost:5002/users/638f5daf5561b563b87b7577/bookmarks/635c50ea9567e5d3fbe978b5
delete bookmark: http://localhost:5002/users/638f5daf5561b563b87b7577/bookmarks/635c50ea9567e5d3fbe978b5

**Pull request merged**